src/google/protobuf/port_def.inc

```diff
@@ -940,11 +940,13 @@ static_assert(PROTOBUF_ABSL_MIN(20230125, 3),
 #pragma warning(disable: 4125)
 #endif
 
+#ifdef PROTOBUF_ENABLE_DEBUG_LOGGING_MAY_LEAK_PII
 #if PROTOBUF_ENABLE_DEBUG_LOGGING_MAY_LEAK_PII
```

Comment:
Let's just get rid of this one, we never define this anyway when we aren't using it